src/layouts/PLayoutGrid/PLayoutGrid.vue

```diff
@@ -0,0 +1,110 @@
+<template>
+  <div class="p-layout-grid">
+    <div class="p-layout-grid__header">
+      <slot name="header">
+        <div class="p-layout-grid__header-leading">
+          <slot name="header-leading" />
+        </div>
+        <div class="p-layout-grid__header-trailing">
+          <slot name="header-trailing" />
+        </div>
+      </slot>
+    </div>
+    <div class="p-layout-grid__toolbar">
+      <slot name="toolbar">
+        <div class="p-layout-grid__toolbar-leading">
+          <slot name="toolbar-leading" />
+        </div>
+        <div class="p-layout-grid__toolbar-trailing">
+          <slot name="toolbar-trailing" />
+        </div>
+      </slot>
+    </div>
+    <main class="p-layout-grid__main">
+      <div v-if="$slots.leftRail" class="p-layout-grid__left-rail">
+        <slot name="leftRail" />
+      </div>
+      <div class="p-layout-grid__content">
+        <slot />
+      </div>
+      <div v-if="$slots.rightRail" class="p-layout-grid__right-rail">
+        <slot name="rightRail" />
+      </div>
+    </main>
+  </div>
+</template>
+
+ <style>
+  .p-layout-grid {
+    @apply
+    w-full
+    h-full
+    border-l
+    flex
+    flex-col
+    border-[color:var(--p-color-divider)];
```

Comment:
you should be able to do this instead for all of these:
```suggestion
    border-divider
```